### ISSUE TITLE
D690 Geometry Inconsistency

### ISSUE THREAD

**Issue by hluszoke** [UCL/STIR-GATE-Connection#94]
In the STIR parameters for the GE D690, the axial size of a ring is defined as 6.54 mm:
Distance between rings (cm)              := 0.654

However, the axial size in the GE D690 GATE macro is only 6.3 mm:
/gate/crystal/geometry/setZLength                  6.3 mm

/gate/crystal/cubicArray/setRepeatVector                  0.0 4.2 6.3 mm

The GATE and STIR geometries seem to be inconsistent with one another for this scanner.

**Comment by KrisThielemans** [comment]
Indeed, the actual D690 will probably have a small gap between blocks, which you could put in GATE. The template.hs file uses the "average ring spacing" we get from GE. If you'd do put in the gaps in GATE, your simulation will be a little bit more realistic (really you'd need to put some optimal insulator between the blocks presumably, but it likely doesn't matter for the default GATE detection model where the optical photons are not simulated).

A PR implementating this small gap would be appreciated!
